### PR TITLE
Moves integration test at top level with jasmine

### PR DESCRIPTION
**Pull request by flovilmart**
- Use jasmine so toolchain is similar to parse-server
- Replaces bash script with pre-test

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,35 +1,60 @@
 # Contributing to the Parse JavaScript SDK
 We want to make contributing to this project as easy and transparent as possible.
 
-If you're looking to get started, but want to ease yourself into the codebase, look for issues tagged [good first bug](https://github.com/ParsePlatform/Parse-SDK-JS/labels/good%20first%20bug). These are simple yet valuable tasks that should be easy to get started.
+If you're looking to get started, but want to ease yourself into the codebase, look for issues tagged [good first bug](https://github.com/parse-community/Parse-SDK-JS/labels/good%20first%20bug). These are simple yet valuable tasks that should be easy to get started.
```
Comment (dplewis) [inline CONTRIBUTING.md]: There isn't a `good first bug` label there is a `good first task`

https://github.com/parse-community/Parse-SDK-JS/labels/good%20first%20task

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,35 +1,60 @@
...
-## Our Development Process
-Most of our work will be done in public directly on GitHub. There may be changes done through our internal source control, but it will be rare and only as needed.
+## Setting up the project for debugging and contributing:
 
-### `master` is unsafe
-Our goal is to keep `master` stable, but there may be changes that your application may not be compatible with. We'll do our best to publicize any breaking changes, but try to use our specific releases in any production environment.
+### Recommended setup:
+
+* [vscode](https://code.visualstudio.com), the popular IDE.
+* [Jest Extesnsion](https://marketplace.visualstudio.com/items?itemName=Orta.vscode-jest) the Jest extension for vscode to run the tests inline and debug quicky.
```
Comment (dplewis) [inline CONTRIBUTING.md]: Extension*

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -40,27 +65,32 @@ Not all issues are SDK issues. If you're unsure whether your bug is with the SDK
 Details are key. The more information you provide us the easier it'll be for us to debug and the faster you'll receive a fix. Some examples of useful tidbits:
 
 * A description. What did you expect to happen and what actually happened? Why do you think that was wrong?
-* A simple unit test that fails. Refer [here][tests-dir] for examples of existing unit tests. See our [README](README.md#usage) for how to run unit tests. You can submit a pull request with your failing unit test so that our CI verifies that the test fails.
+* A simple unit test that fails. Refer [here][tests-dir] for examples of existing unit tests and [here](integration-test-dir) for integration tests examples. See for how to setup your machine and run unit tests in [this]() guide. You can submit a pull request with your failing unit test so that our CI verifies that the test fails.
```
Comment (dplewis) [inline CONTRIBUTING.md]: Most of the links here are broken...

**Comment by flovilmart** [comment]
Thanks for the review @dplewis, i’ll Fix up before end of week

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,66 +1,97 @@
...
 By contributing to the Parse JavaScript SDK, you agree that your contributions will be licensed under its license.
 
  [google-group]: https://groups.google.com/forum/#!forum/parse-developers
- [stack-overflow]: http://stackoverflow.com/tags/parse.com
- [bug-reports]: https://www.parse.com/help#report
- [rest-api]: https://www.parse.com/docs/rest/guide
+ [stack-overflow]: http://stackoverflow.com/tags/parse-server
+ [bug-reports]: https://github.com/parse-community/parse-server/issues
+ [rest-api]: https://docs.parseplatform.org/rest/guide
  [parse-api-console]: http://blog.parse.com/announcements/introducing-the-parse-api-console/
```
Comment (dplewis) [inline CONTRIBUTING.md]: The link should be http://blog.parseplatform.org/announcements/introducing-the-parse-api-console/

**Comment by dplewis** [comment]
@flovilmart Whats the status here?

**Comment by flovilmart** [comment]
@dplewis that looks all good!

**Comment by dplewis** [comment]
@flovilmart In parse-server you added `npm run watch` would it be possible to add it here. 

When I'm testing I only need to rebuild node, not browser / react. Its kinda time consuming.

**Comment by flovilmart** [comment]
Ah right, I believe that should be easy, as it’s gulp based, with gulp-watch you should be able to craft something easily

**Comment by dplewis** [comment]
Can you add it? 🙏

My knowledge of gulp is a little above zero.

Currently I'm having this error

```
[12:15:37] Starting 'browserify'...
[12:15:38] The following tasks did not complete: browserify
[12:15:38] Did you forget to signal async completion?
```

**Comment by flovilmart** [comment]
Uhm yeah I'll need to have a look, but it's gonna be later today, busy on something else now ;) Would be faster for you to learn gulp :P

**Comment by dplewis** [comment]
I know gulp just wanted you to do it lol

**Pull request merged**